src/Microsoft.ML.HalLearners/ProjectionCatalog.cs

```diff
@@ -1,47 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Transforms.Projections;
-
-namespace Microsoft.ML
-{
-    public static class ProjectionCatalog
```

Comment:
out of curiosity, why?

Reply:
Thy said "one per assembly"  so moved it to the HalLearners catalog. 

---
In reply to: [235497897](https://github.com/dotnet/machinelearning/pull/1682#discussion_r235497897) [](ancestors = 235497897)